tracing/src/main/java/com/palantir/remoting3/tracing/AsyncSlf4jSpanObserver.java

```diff
@@ -16,171 +16,29 @@
 
 package com.palantir.remoting3.tracing;
 
-import com.fasterxml.jackson.annotation.JsonInclude;
-import com.fasterxml.jackson.core.JsonProcessingException;
-import com.fasterxml.jackson.databind.ObjectMapper;
-import com.fasterxml.jackson.databind.annotation.JsonSerialize;
-import com.google.common.collect.Lists;
-import com.palantir.logsafe.UnsafeArg;
 import com.palantir.remoting.api.tracing.Span;
-import com.palantir.remoting3.ext.jackson.ObjectMappers;
-import java.net.Inet4Address;
-import java.net.Inet6Address;
+import com.palantir.remoting.api.tracing.SpanObserver;
+import com.palantir.tracing.AsyncSpanObserver;
 import java.net.InetAddress;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
 import java.util.concurrent.ExecutorService;
-import org.immutables.value.Value;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 /**
  * An {@link AsyncSpanObserver asynchronous SpanObserver} that logs every span to a configurable SLF4J {@link Logger}
  * with log-level {@link Logger#info INFO}. Logging is performed asynchronously on a given executor service.
  */
-public final class AsyncSlf4jSpanObserver extends AsyncSpanObserver {
-    private static final ObjectMapper mapper = ObjectMappers.newClientObjectMapper();
+public final class AsyncSlf4jSpanObserver implements SpanObserver {
 
-    private final Logger logger;
-    private final ZipkinCompatEndpoint endpoint;
+    private final com.palantir.tracing.AsyncSlf4jSpanObserver delegate;
 
-    @JsonSerialize(as = ImmutableZipkinCompatSpan.class)
-    @Value.Immutable
-    @Value.Style(visibility = Value.Style.ImplementationVisibility.PACKAGE)
-    abstract static class ZipkinCompatSpan {
```

Comment:
These classes have been completely deleted because they're an implementation detail that now exists in the new tracing-java class: `com.palantir.tracing.AsyncSlf4jSpanObserver`